ui/app/src/model/config-client.ts

```diff
@@ -135,6 +135,13 @@ export interface ExplorerConfig {
   enable: boolean;
 }
 
+export type Severity = 'warning' | 'info' | 'error' | 'success';
+
+export interface Banner {
+  severity?: Severity;
+  information: string;
```

Comment:
you change `information` by `message` in the Go code but you didn't reflect this change in the frontend code. So in the current status, this code does not work at all.

It would be great to test the changes you would like to provide.